pwiz_tools/Skyline/Model/Results/PeakIntegrator.cs

```diff
@@ -104,6 +118,59 @@ private ChromPeak IntegratePeakWithoutBackground(float startTime, float endTime,
             return new ChromPeak(RawTimeIntensities ?? InterpolatedTimeIntensities, startTime, endTime, flags);
         }
 
+        /// <summary>
+        /// If the Acquisition Method is DDA, then change the Area of the peak to
+        /// be the intensity at the time point with the highest total MS1 intensity
```

Comment:
The code looks like you are using highest total MS2 intensity. So, I think this comment is incorrect.